zng/bytes.go

```diff
@@ -18,7 +18,7 @@ func EncodeBytes(b []byte) zcode.Bytes {
 
 func DecodeBytes(zv zcode.Bytes) ([]byte, error) {
 	if zv == nil {
-		return nil, ErrUnset
+		return nil, nil
 	}
 	return []byte(zv), nil
```

Comment:
Nit: No need for the if statement; `return []byte(zv), nil` alone is sufficient.